### PR TITLE
COMMON: correct rc handling with s390_pcc function

### PR DESCRIPTION
**Pull request by jschmidb**
Let s390_pcc return the condition code of the PCC instruction. Also fix the error handling in the calling functions.

### CODE REVIEW

File: usr/lib/common/pkey_utils.c [hunk 1]
```diff
@@ -234,27 +234,30 @@ static int s390_kmac(unsigned long func, void *param,
...
-
-    __asm__ volatile (
-        "0: .long   %[opc] << 16\n"
-        "   brc 1,0b\n"
-        :
-        : [fc] "d" (r0), [param] "a" (r1), [opc] "i" (0xb92c)
-        : "cc", "memory");
-
-    return 0;
+	register unsigned long r0 __asm__("0") = (unsigned long)func;
```
Comment (ifranzki) [inline usr/lib/common/pkey_utils.c:246]: Please use OCK coding style, i.e. 4 blanks instead of tabs.

Comment (jschmidb) [reply usr/lib/common/pkey_utils.c]: Fixed.

**Comment by jschmidb** [comment]
Yes, I ran the aes tests on ep11 token with 'PKEY_MODE ENABLE4ALL', and on CCA token with 'PKEY_MODE = ENABLED'. I got calls to s390_pcc from CCA when performing the xts tests. Cannot say why I'm not getting calls from the cmac tests in both tokens.

**Comment by jschmidb** [comment]
> But for EP11, I guess you should get calls to s390_pcc for CMAC.

I'm getting the calls now! It was a config problem on my test system.

**Pull request merged**